### PR TITLE
[cmake][win32][fix] Don't empty path, prepend our libs to the start of it

### PR DESCRIPTION
**Pull request by Paxxi**
empting the path caused new exciting errors, this should hopefully work
## Types of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the [Code guidelines](https://codedocs.xyz/xbmc/xbmc/code_guidelines.html) of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [ ] I have read the [CONTRIBUTING](https://github.com/xbmc/xbmc/blob/master/CONTRIBUTING.md) document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

### CODE REVIEW

**Comment by Paxxi** [comment]
jenkins build this please

File: cmake/scripts/windows/ArchSetup.cmake [hunk 1]
```diff
@@ -22,7 +22,8 @@ set(PYTHON_INCLUDE_DIR ${CMAKE_SOURCE_DIR}/project/BuildDependencies/include/pyt
 # Emptying PATH so that system installed libraries (MySql, Python) are not picked up.
 # The VS/bin directory has to be in PATH for example for NMake.
 get_filename_component(TOOLCHAIN_PATH ${CMAKE_CXX_COMPILER} DIRECTORY)
```
Comment (hudokkow) [inline cmake/scripts/windows/ArchSetup.cmake]: If this works as intended, line above can go.

Comment (Paxxi) [inline cmake/scripts/windows/ArchSetup.cmake]: oops, missed that line, thanks

**Comment by Paxxi** [comment]
jenkins build this please

**Pull request closed without merging**